File: C/code/is_prime.c
# include <stdio.h>
int is_prime (int num) {
	int b = num - 1;
	while (b > 1) {
		if (num % b == 0) {
			return 0;
		}
		b--;
	}
	return 1;
}

int main() {
	int x;
	int results;
	printf("User, enter a number: ");
	scanf("%d", &x);
	results = is_prime(x);
	if (results == 1) {
		printf("%d IS a prime\n", x);
	}
	else {
		printf("%d is NOT a prime\n", x);
	}
}

File: C#/code/is_prime.cs
namespace aviv_yunker {
	class is_prime {
		public static void Main (string [] args) {
			string x;
			int num, b, answer = 1;
			System.Console.WriteLine("User, enter a number: ");
			x = System.Console.ReadLine();
			num = int.Parse(x);
			b = num - 1;
			while (b > 1) {
				if (num % b == 0) {
					answer = 0;
					break;
				}
				b--;
			}
			if (answer == 0) {
				System.Console.WriteLine("{0} is NOT a prime", x);
			}
			else {
				System.Console.WriteLine("{0} IS a prime", x);
			}
		}
	}
}
